app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala

```diff
@@ -248,13 +251,16 @@ class MovementsRepositoryImpl @Inject() (
   def getMovements(
     eoriNumber: EORINumber,
     movementType: MovementType,
-    updatedSince: Option[OffsetDateTime]
+    updatedSince: Option[OffsetDateTime],
+    movementEORI: Option[EORINumber]
   ): EitherT[Future, MongoError, Option[NonEmptyList[MovementWithoutMessages]]] = {
     val selector: Bson = mAnd(
       mEq("enrollmentEORINumber", eoriNumber.value),
       mEq("movementType", movementType.value),
-      mGte("updated", updatedSince.map(_.toLocalDateTime).getOrElse(EPOCH_TIME))
+      mGte("updated", updatedSince.map(_.toLocalDateTime).getOrElse(EPOCH_TIME)),
```

Comment:
Why is this filter in the greater than or equal to updated? Can't we just add a mEq("movementEORINumber", movementEORI.getOrElse(empty())) in the mAnd list?